app/src/components/Slider.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useId } from 'react'
+
+const Slider = ({ maxRange }: { maxRange: number }) => {
```

Comment:
Could you move the prop to its own interace?

Reply:
Instead of just `maxRange`, I would have the following properties:
`maxValue`, `minValue`, `initialValue`